components/esp_websocket_client/include/esp_websocket_client.h

```diff
@@ -155,16 +155,36 @@ esp_websocket_client_handle_t esp_websocket_client_init(const esp_websocket_clie
 esp_err_t esp_websocket_client_set_uri(esp_websocket_client_handle_t client, const char *uri);
 
 /**
- * @brief      Set additional websocket headers for the client, when performing this behavior, the headers will replace the old ones
- * @pre        Must stop the WebSocket client before set headers if the client has been connected
+ * @brief      Sets additional WebSocket headers for the client
+ * @pre        The WebSocket client must be stopped before setting headers if it is already connected.
  *
- * @param[in]  client  The client
- * @param headers  additional header strings each terminated with \r\n
+ *  Notes:
+ *  - When setting headers using this method, any previous headers will be replaced.
+ *    To set multiple headers, you can concatenate them using the "\r\n" delimiter.
+ *    Example: websocket_cfg.headers = "Sec-WebSocket-Key: my_key\r\nPassword: my_pass\r\n";
```

Comment:
Also this is an example of setting the headers as a part of config option. Should we clearly say what are the options that users have? For example, say you can:
1) Set headers directly in the config options (each item terminated by [CR][LF])
2) Use this API, (which will overwrite (1), ... )
3) Use the *append* API, which adds a single header